lib/event_tracker.rb

```diff
@@ -1,3 +1,6 @@
+require 'require_all'
+require_all 'lib'
```

Comment:
Are we adding a 500LOC extra dependency to avoid writing something like:
```ruby
Dir[File.join('lib', '**', '*.rb')].each{ |lib| require lib }
```
or there's is additional advantages, future planned use that I am yet unaware of?